doc/ref_cert/README.md

```diff
@@ -6,3 +6,4 @@
 
 ## Available Programs
```

Comment:
@xmulligan can you also please edit RC2 chapter owenership in CODEOWNERS the same way other WS are defined in there?